pkg/rest/server.go

```diff
@@ -98,13 +98,13 @@ func (s *Server) Run() {
 }
 
 func (s *Server) Stop(timeout time.Duration) {
-	s.logger.Info("Server is going to sleep 10 seconds before shutting down")
+	s.logger.Info("Health check server is going to sleep 1 seconds before shutting down")
```

Comment:
The rest server is a common package that can be used for any service. So by using the `Health check server` in the log, it seems this package is for health checking only. It could confuse us.

Maybe `Rest server is going to sleep 1 second before shutting down` would be better.